src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif, Arial;
+  font-weight: 400;
+  font-style: normal;
+  font-size: 12px;
+}
+
+header {
+  margin: 10px 0 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo-link {
+  text-align: center;
+}
+
+a.is-active {
+  color: #00acdc;
+  text-decoration: underline;
+  position: relative;
+}
```

Comment:
The selector `a.is-active` is a bit too general. It will apply styles to any link with the class `is-active`, not just the ones in the navigation. To avoid potential conflicts, you should make your selector more specific, for example `.nav__link.is-active`.